scripts/e2e-servers

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env node
+const moduleManager = require('../tests/protractor/module-manager');
+
+Promise.resolve()
+  .then(() => moduleManager.startNodeModule('api'))
+  .then(() => moduleManager.startNodeModule('sentinel'));
```

Comment:
I think because you're not passing `startOutput` the `api` promise will never resolve so sentinel will never run.

It'd also be nice to output something when api and sentinel are running.